.github/workflows/ci.yml

```diff
@@ -167,20 +167,17 @@ jobs:
                   restore-keys: ${{ runner.os }}-composer-
 
             - name: Install dependencies
-              run: composer install --prefer-dist
+              run: composer update --prefer-dist
 
             - name: Run PHPStan
               run: composer run-script phpstan
 
     spellchecking:
         name: Spellchecking
-        runs-on: ubuntu-latest
-        strategy:
-            fail-fast: true
```

Comment:
`fail-fast` does nothing without a matrix.

Reply:
Ah yes, I thought so but was too lazy too look it up when adding the spellcheck workflow.